packages/appium/scripts/postinstall.js

```diff
@@ -14,20 +17,34 @@ async function main () {
   try {
     extension = require('../build/lib/cli/extension');
   } catch (e) {
-    throw new Error(`Could not load extension CLI file; has the project been transpiled? ` +
-                    `(${e.message})`);
+    throw new Error(
+      `Could not load extension CLI file; has the project been transpiled? ` + `(${e.message})`
+    );
   }
 
-  const {DEFAULT_APPIUM_HOME, DRIVER_TYPE, PLUGIN_TYPE} = require('./build/lib/extension-config');
+  const {DRIVER_TYPE, PLUGIN_TYPE} = require('../build/lib/constants');
+  const {loadExtensions} = require('../build/lib/extension');
   const {runExtensionCommand} = extension;
-  const appiumHome = process.env.npm_config_appium_home || DEFAULT_APPIUM_HOME;
-  const specs = [[DRIVER_TYPE, driverEnv], [PLUGIN_TYPE, pluginEnv]];
+  const specs = [
+    [DRIVER_TYPE, driverEnv],
+    [PLUGIN_TYPE, pluginEnv],
+  ];
+
+  const appiumHome = process.env.npm_config_appium_home || (await env.resolveAppiumHome());
```

Comment:
shouldn't `npm_config_appium_home` be also considered in `resolveAppiumHome` call?

Reply:
I'm not sure what you're asking.  Are you asking if `resolveAppiumHome()` should consider that environment variable in addition to `APPIUM_HOME`?

Reply:
yes, exactly

Reply:
i think we settled on just leaning on APPIUM_HOME. i think this code came from a time when we had the `--appium-home` command line flag, and it mirrors that. but we no longer have that flag.